README.md

```diff
@@ -11,3 +11,16 @@ This Repository
 ===============
 
 This repository contains the markup needed to deploy the floeproject.org website.
+
+
+News Item Template
+==================
+
+An html template is provided for the news item here: news/YYYY-MM-DD-news-item-title.html.
```

Comment:
For clarity, change news/YYYY-MM-DD-news-item-title.html to \`./news/YYYY-MM-DD-news-item-title.html\`.